nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -267,6 +271,15 @@ in
       '';
     };
 
+    enableReload = mkOption {
```

Comment:
If this is enabled and there is an error in the new config file, will error be reported?  Related https://github.com/NixOS/nixpkgs/issues/73455

Reply:
Good point, I just tested it and Caddy reports an error in the journal and then exits. I’ve added an execution of the validate command on reload. Now any error in the configuration is correctly reported:

```
reloading the following units: caddy.service
Job for caddy.service failed.
See "systemctl status caddy.service" and "journalctl -xeu caddy.service" for details.
warning: error(s) occurred while switching to the new configuration
```